fil-proofs-tooling/scripts/retry.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+# This software is freely available under the Apache 2.0 license.
+# Full license available here: https://www.apache.org/licenses/LICENSE-2.0.txt
```

Comment:
no need for license header, this repo is already covered under mit/apache2 dual license